Apps/HeadlessScreenshotApp/CMakeLists.txt

```diff
@@ -6,8 +6,8 @@ FetchContent_MakeAvailable(CMakeExtensions)
 FetchContent_MakeAvailable_With_Message(DirectXTK)
 
 set_property(TARGET DirectXTK PROPERTY UNITY_BUILD false)
-set_property(TARGET DirectXTK PROPERTY FOLDER Dependencies/DirectXTK)
-set_property(TARGET xwbtool PROPERTY FOLDER Dependencies/DirectXTK)
+set_property(TARGET DirectXTK PROPERTY FOLDER Apps/Dependencies/DirectXTK)
+set_property(TARGET xwbtool PROPERTY FOLDER Apps/Dependencies/DirectXTK)
```

Comment:
This shouldn't be here at all. See
https://github.com/BabylonJS/BabylonNativeExamples/blob/f0c68910c1d6b8454e323e3e36476c0c0cf79fcd/Dependencies/CMakeLists.txt#L4-L9